app/services/puppet_fact_parser.rb

```diff
@@ -251,8 +251,12 @@ def os_release
       return "99" if distro_codename =~ /sid/
       release = os_release_full
       case release
-      when 'bullseye/sid' # Debian Bullseye testing will be 11
-        '11'
+      when 'bookworm/sid' # Debian Bookworm will be 12
```

Comment:
I'd expect line 251 to already match. Perhaps that line can go away now?

Reply:
Welcome to the last 30 minutes of my life.

Line 251 matches on `distro_codename` which is `facts[os][distro][codename]` which on Sid *today* returns `bookworm` (but did in the past contain 'sid', which is what the match is based on), so it doesn't match.
Line 252+ match on `os_release_full`, which is `facts[os][release][full]` which on Sid today returns `bookworm/sid` and that gets matched in 254.

I suppose we could drop Line 251 for now (as today testing/bookworm and unstable/sid are indistinguishable from a Puppet perspective), but we could also keep it if the difference is ever coming back.